lib/src/transformer/impl_generation.dart

```diff
@@ -88,6 +88,15 @@ class ImplGenerator {
       // We can safely make this abstract, since we already have a runtime warning when it's
       // instantiated.
       if (!declarations.props.node.isAbstract) {
+
+        // Remove the with clause from the public props class signature for dart 2 builder compatibility.
+        // In Dart 2 there will be a dual class set up requiring a private and public props class. The private
+        // annotated class defines the interface via uninitialized fields. The public un-annotated class is a
+        // stub and needs to be transformed to include the concrete accessor implementations.
```

Comment:
This comment should explicitly explain why we're removing the with clause. Maybe something like this:

> The public props class signature includes a `with <PropsClass>AccessorsMixin` clause for dart 2 builder compatibility. But in Dart 1, the transformer is able to generate the concrete accessors inline without a separate mixin. For this reason, the transformer first removes this with clause and then generates the concrete accessors.